File: Cpp/Ram.cpp
class Ram {

    /* Private Attribute */
    private:

        string capacity;
        int price;

    /* Public Attribute and Method */
    public:

        /* Constructor */
        Ram() {
            this->capacity = "Capacity";
            this->price = 0;
        }

        Ram(string capacity, int price) {
            this->capacity = capacity;
            this->price = price;
        }

        /* Setter dan Getter Capacity */
        void setCapacity(string capacity) {
            this->capacity = capacity;
        }

        string getCapacity() {
            return this->capacity;
        }

        /* Setter dan Getter Price */
        void setPrice(int price) {
            this->price = price;
        }

        int getPrice() {
            return this->price;
        }

        /* Destructor */
        ~Ram() {

        }

};

File: php/Ram.php
<?php

class Ram {
    
    /* Private Attribute */
    private $capacity = "Capacity";
    private $price = 0;

    /* Constructor */
    public function __construct($capacity = "Capacity", $price = 0) {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    /* Setter dan Getter Capacity */
    public function setCapacity($capacity) {
        $this->capacity = $capacity;
    }

    public function getCapacity() {
        return $this->capacity;
    }

    /* Setter dan Getter Price */
    public function setPrice($price) {
        $this->price = $price;
    }

    public function getPrice() {
        return $this->price;
    }

    /* Destructor */
    public function __destruct() {

	}

}

?>